R/g.part3.R

```diff
@@ -147,6 +147,7 @@ g.part3 = function(metadatadir = c(), f0, f1, myfun = c(),
                                 varlist = c(unclass(lsf.str(envir = asNamespace("GGIR"), all = T)),
                                             "MONITOR", "FORMAT"),
                                 envir = as.environment(asNamespace("GGIR")))
+        parallel::clusterEvalQ(cl, Sys.setlocale("LC_TIME", "C"))
```

Comment:
nice! Maybe part 6 too? Or can this definitely never apply to part 6?

Reply:
Yes, but I have now committed that to a different open PR which comes with many other updates to part6, in that way I the merge will go easier [16b97f0](https://github.com/wadpac/GGIR/pull/1028/commits/16b97f05ca0740851ffd3583230f161cacd9da42)